docs/nillion-devnet.md

```diff
@@ -1,6 +1,6 @@
-# run-local-cluster
+# nillion-devnet
```

Comment:
Review this file, there are some references to cluster that should be changed to devnet